src/foam/u2/view/SummaryView.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * @license
+ * Copyright 2022 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.u2.view',
+  name: 'SummaryView',
```

Comment:
Why not use CitationView?

Reply:
because it's fun to see you state that as a question (I fixed it)

Reply:
CitationView is more full featured.  But looking back now, SummaryView would have been a good choice.  Also since CitationView is faceted. 
So @KernelDeimos do you need a simpler view for toSummary?

Reply:
I could've sworn I pushed a change to this branch to use CitationView. CitationView works but I guess I have to do it again